packages/server/graphql/private/queries/user.ts

```diff
@@ -5,7 +5,7 @@ const user: QueryResolvers['user'] = async (_source, {email, userId}, {dataLoade
   if (email) {
     return getUserByEmail(email)
   }
-  return dataLoader.get('users').load(userId)
+  return (userId && (await dataLoader.get('users').load(userId))) || null
```

Comment:
the way it's written it also handles if the dataloader returns `undefined`.
To fix, we should probably make the User dataloader return `User | null` instead of `User | undefined`.
The weird thing is, I don't really see where it's getting `undefined` from 😕